src/Tasks/Microsoft.NET.Build.Tasks.NETFramework/Microsoft.NET.Build.Tasks.NETFramework.csproj

```diff
@@ -0,0 +1,111 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+    <PackageId>Microsoft.NET.Sdk</PackageId>
+    <Description>The MSBuild targets and properties for building .NET Core projects.</Description>
+    <TargetFrameworks>netcoreapp1.0;net46</TargetFrameworks>
+    <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">netcoreapp1.0</TargetFrameworks>
+    <GenerateDependencyFile>false</GenerateDependencyFile>
+    <PackageOutputPath>$(OutputPath)Packages\</PackageOutputPath>
+    <PackageLayoutOutputPath>$(OutputPath)Sdks\$(PackageId)\</PackageLayoutOutputPath>
+    <OutputPath>$(PackageLayoutOutputPath)tools\</OutputPath>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+    <NoPackageAnalysis>true</NoPackageAnalysis>
+    <EnableDefaultItems>false</EnableDefaultItems>
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
+  </PropertyGroup>
+
+  <ItemDefinitionGroup>
+    <PackageReference>
+      <PrivateAssets>All</PrivateAssets>
+      <Publish>true</Publish>
+    </PackageReference>
+  </ItemDefinitionGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Build.Framework" Version="$(MsBuildPackagesVersion)" ExcludeAssets="Runtime" />
+    <PackageReference Include="Microsoft.Build.Utilities.Core" Version="$(MsBuildPackagesVersion)" ExcludeAssets="Runtime" />
+    <PackageReference Include="System.Reflection.Metadata" Version="$(SystemReflectionMetadataVersion)" Condition=" '$(TargetFramework)' == 'netcoreapp1.0' " />
+    <PackageReference Include="System.Diagnostics.FileVersionInfo" Version="4.0.0" Condition=" '$(TargetFramework)' == 'netcoreapp1.0' " />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Compile Include="..\Microsoft.NET.Build.Tasks\BuildErrorException.cs">
+      <Link>BuildErrorException.cs</Link>
+    </Compile>
+    <Compile Include="..\Microsoft.NET.Build.Tasks\ConflictResolution\*.cs">
+      <Link>ConflictResolution\%(FileName)%(Extension)</Link>
+    </Compile>
+    <Compile Include="..\Microsoft.NET.Build.Tasks\DiagnosticsHelper.cs">
+      <Link>DiagnosticsHelper.cs</Link>
+    </Compile>
+    <Compile Include="..\Microsoft.NET.Build.Tasks\DiagnosticMessageSeverity.cs">
+      <Link>DiagnosticMessageSeverity.cs</Link>
+    </Compile>
+    <Compile Include="..\Microsoft.NET.Build.Tasks\ItemUtilities.cs">
+      <Link>ItemUtilities.cs</Link>
+    </Compile>
+    <Compile Include="..\Microsoft.NET.Build.Tasks\GetDependsOnNETStandard*.cs">
+      <Link>%(FileName)%(Exetnsion)</Link>
```

Comment:
Typo: `Exetnsion` (but as mentioned previously, this shouldn't be needed at all, at least once the update flows to the right place)